lib/roo/excelx/cell/base.rb

```diff
@@ -71,9 +71,14 @@ def to_s
           formatted_value
         end
 
-        # DEPRECATED: Please use link instead.
+        # DEPRECATED: Please use link? instead.
         def hyperlink
-          warn '[DEPRECATION] `hyperlink` is deprecated.  Please use `link` instead.'
+          warn '[DEPRECATION] `hyperlink` is deprecated.  Please use `link?` instead.'
+        end
+
+        # DEPRECATED: Please use link? instead.
+        def link
+          warn '[DEPRECATION] `link` is deprecated.  Please use `link?` instead.'
```

Comment:
We should also return expected value with a deprecation warning. 
Which should be `link?`

But I think storing a boolean value to `@link` was a bug.

if we consider previous implementation of link buggy then link method must be as follow:
```
def link
  link? ? value : nil
end
```
or it should be as follow:

```
def link
  warn '[DEPRECATION] `link` is deprecated.  Please use `link?` instead.'
  link?
end
```